pkg/blockstorage/azure/client.go

```diff
@@ -20,11 +20,13 @@ package azure
 
 import (
 	"context"
+	"github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/go-autorest/autorest/azure/auth"
```

Comment:
Looks like VSCode added these back. The client code could still be using these libraries.